port_forwarding_test.go

```diff
@@ -22,101 +22,95 @@ const (
 )
 
 func TestExposeHostPorts(t *testing.T) {
-	tests := []struct {
-		name          string
-		numberOfPorts int
-		hasNetwork    bool
-		hasHostAccess bool
-	}{
-		{
-			name:          "single port",
-			numberOfPorts: 1,
-			hasHostAccess: true,
-		},
-		{
-			name:          "single port using a network",
-			numberOfPorts: 1,
-			hasNetwork:    true,
-			hasHostAccess: true,
-		},
-		{
-			name:          "multiple ports",
-			numberOfPorts: 3,
-			hasHostAccess: true,
-		},
-		{
-			name:          "single port with cancellation",
-			numberOfPorts: 1,
-			hasHostAccess: false,
+	t.Run("single-port", func(t *testing.T) {
+		testExposeHostPorts(t, 1, false, true)
+	})
+
+	t.Run("single-port-using-network", func(t *testing.T) {
+		testExposeHostPorts(t, 1, true, true)
+	})
+
+	t.Run("single-port-cancellation", func(t *testing.T) {
+		testExposeHostPorts(t, 1, false, false)
```

Comment:
question: does it make sense another test case for "true, false" (has network but no host access) or that combination is out of the scope?